controllers/nutanixmachine_controller.go

```diff
@@ -447,18 +453,17 @@ func (r *NutanixMachineReconciler) reconcileNode(rctx *nctx.MachineContext) (rec
 	}
 
 	// Retrieve the remote node
-	nodeName := rctx.NutanixMachine.Name
+	nodeName := rctx.Machine.Name
 	node := &corev1.Node{}
 	nodeKey := apitypes.NamespacedName{
 		Namespace: "",
 		Name:      nodeName,
 	}
 
-	for {
-		err = remoteClient.Get(rctx.Context, nodeKey, node)
-		if err == nil {
-			break
-		}
+	// Does this loop only run once?
```

Comment:
which loop? is comment valid?

Reply:
Ended up removing the loop if you look at changes side by side. Will update comment.